pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -1028,3 +1023,19 @@ func (l *LibvirtDomainManager) updateVMIMigrationMode(mode v1.MigrationMode) {
 	})
 	log.Log.V(4).Infof("Migration mode set in metadata: %s", l.metadataCache.Migration.String())
 }
+
+func shouldConfigureParallelMigration(options *cmdclient.MigrationOptions) (shouldConfigure bool, threadsCount int) {
+	if options == nil {
+		return
+	}
+	if options.AllowPostCopy {
```

Comment:
So, multifd is enabled and set to 8 when post-copy is not enabled and CPU limits are not set on the VM, right?
Probably worth a line or 2 in https://kubevirt.io/user-guide/compute/live_migration/

Reply:
> So, multifd is enabled and set to 8 when post-copy is not enabled and CPU limits are not set on the VM, right? 

Yes, that's right!

> Probably worth a line or 2 in [kubevirt.io/user-guide/compute/live_migration](https://kubevirt.io/user-guide/compute/live_migration/)

TBH I'm conflicted on that. As written above, I see this as an implementation detail that should be transparent for users. In which cases do you think a user would be interested in the number of multifd threads being set?